tables/automl/automl_tables_model.py

```diff
@@ -116,19 +118,28 @@ def list_models(project_id, compute_region, filter_=None):
         print("Model id: {}".format(model.name.split("/")[-1]))
         print("Model display name: {}".format(model.display_name))
         metadata = model.tables_model_metadata
-        print("Target column display name: {}".format(
-            metadata.target_column_spec.display_name))
-        print("Training budget in node milli hours: {}".format(
-            metadata.train_budget_milli_node_hours))
-        print("Training cost in node milli hours: {}".format(
-            metadata.train_cost_milli_node_hours))
+        print(
```

Comment:
This looks weird. Why the `print(` on its own line instead of minimizing the vertical space used?